### PR TITLE
add haml and fix ruby version

### PR DESCRIPTION
**Pull request by helphop**
I had to fix the version of ruby as there were no 2.4.0 versions available.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.4.0
```
Comment (josh-m-sharpe) [inline .ruby-version]: I don't think this is true - have you tried updating your install of rbenv?  2.4.0 has been out for several months now - there has to be a valid version of it available.

https://www.ruby-lang.org/en/news/2016/12/25/ruby-2-4-0-released/

**Comment by josh-m-sharpe** [comment]
The rest of this looks good though.  Take a look at the version thing and see about updating that part of the PR.

How did the authlogic stuff go?

**Comment by helphop** [comment]
Hi Josh,

I did an update of my rbenv and the versions it shows that are available
are 2.4.0-dev and 2.4.0-prev  I will try again and report back.

I have a question about using git in a team like this.  I created a branch
for the HAML gem and pushed it.

Should I just create a branch for my To-do's and not ones specific for each
To-do?


Thanks

Mitchell

Mitch

File: .ruby-version (hunk 1 above)
Comment (helphop) [inline .ruby-version]: I've updated rbenv now two times and I get the same listing for ruby 2.4.0 as follows:
  2.4.0-dev
  2.4.0-preview1
  rbx-2.4.0
What can I do?

Comment (helphop) [inline .ruby-version]: Until I fix this conflict I can't proceed

**Comment by josh-m-sharpe** [comment]
21924dc6e856ebcdb30a0eecead6de34a6afe227

**Pull request closed without merging**